README.md

```diff
@@ -37,14 +37,11 @@ You can install this program globally with `npm i -g zksync-cli` or run the comm
 
 - `zksync-cli --version`: Returns the current version
 
-- `zksync-cli localnet`: Manages a local zkSync Era and Ethereum L1 testnet (it requires docker running on your system). It supports a set of sub-subcommands:
-  - `zksync-cli localnet up`: Bootstrap L1 and L2 localnets.
-  - `zksync-cli localnet down`: clear L1 and L2 localnets.
-  - `zksync-cli localnet start`: start L1 and L2 localnets.
-  - `zksync-cli localnet stop`: stop L1 and L2 localnets.
-  - `zksync-cli localnet logs`: Display logs.
-  - `zksync-cli localnet help`: Display this message and quit.
-  - `zksync-cli localnet wallets`: Display seeded wallet keys.
+- `zksync-cli local` - All-in-one tool for local zkSync development. It supports a set of sub-subcommands:
+  - `zksync-cli local start` - start local setup (will ask to configure if starting for the first time)
+  - `zksync-cli local stop` - stop currently running modules
+  - `zksync-cli local clean` - clean data for all modules
+  - `zksync-cli local config` - setup new config for local setup (select modules)
 
```

Comment:
Thoughts on including `zksync-cli local logs` cmd which will show Docker logs accordingly?

Reply:
I think this would be quite handy to have especially for the dockerized setup where you have to wait for the contracts to be deployed to L1 chain. I personally like to follow so I know when its completed.

Reply:
Yeah, definitely need this command. Will create new task and will work on this feature in another PR